app/utils/guaranteeParsing/RouteChecker.scala

```diff
@@ -31,15 +31,13 @@ class RouteChecker @Inject()(xmlReaders: GuaranteeXmlReaders) {
         xmlReaders.officeOfDestination(xml) match {
           case Left(error) => Left(error)
           case Right(destination) => {
-            if(departure.value.startsWith("GB")){
-              Right(destination.value.startsWith("GB"))
-            }
-            else {
-              Left(InappropriateDepartureOffice("Inappropriate Departure Office"))
+            departure.prefix match {
+              case "GB" => Right(destination.value.startsWith("GB"))
+              case "XI" => Right(false)
```

Comment:
Did we decide that there are no invalid ports, anything that's not GB or XI will go to GB?  Did I dream it?

Reply:
I've no point of reference. This only focuses on the departure office, and we don't handle anything that isn't XI or GB if that's what you're questioning.

If you're thinking about arrivals, that isn't relevant to the routechecker.